gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Aug 31 19:13:22 PDT 2016
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-all.zip
+#Wed Sep 21 15:07:15 AEST 2016

```

Comment:
Again, commit contents should be apparent from description. Updating the gradle wrapper should not be part of it.